cmake/common.cmake

```diff
@@ -487,12 +487,19 @@ function(add_perf_test)
     set(LABEL_ARG "${PARSED_ARGS_NAME}_${TESTS_SUFFIX}^")
   endif()
 
+  if(PERF)
+    set(NODES -n 10.0.0.5 -n 10.0.0.8 -n 10.0.0.11)
```

Comment:
Is there a way we can avoid having the private IPs of our "slave" machines hardcoded here?

Reply:
There's a bunch of different schemes, but they all involve defining the hosts somewhere. If you prefer something else, I'm happy to change it.